src/main/java/view/InputView.java

```diff
@@ -0,0 +1,30 @@
+package view;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    public static final String CAR_GUIDE = "자동차 대수는 몇 대 인가요?";
+    public static final String ROUND_GUIDE = "시도할 회수는 몇 회 인가요?";
+
```

Comment:
```suggestion
    public static final String CAR_GUIDE = "자동차 대수는 몇 대 인가요?";
    public static final String ROUND_GUIDE = "시도할 회수는 몇 회 인가요?";

```
자바 코딩 컨벤션에 따라 상수 선언은 class 선언 바로 밑에 정의해주세요!